src/test/java/uk/gov/hmcts/reform/em/orchestrator/service/ccdapi/CCDUpdateServiceTest.java

```diff
@@ -0,0 +1,119 @@
+package uk.gov.hmcts.reform.em.orchestrator.service.ccdapi;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.jupiter.api.Test;
+import org.mockito.ArgumentCaptor;
+import uk.gov.hmcts.reform.authorisation.generators.AuthTokenGenerator;
+import uk.gov.hmcts.reform.ccd.client.CoreCaseDataApi;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDataContent;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+import uk.gov.hmcts.reform.ccd.client.model.CaseResource;
+import uk.gov.hmcts.reform.ccd.client.model.Event;
+import uk.gov.hmcts.reform.ccd.client.model.StartEventResponse;
+import uk.gov.hmcts.reform.em.orchestrator.service.ccdcallbackhandler.CcdCallbackDto;
+import uk.gov.hmcts.reform.em.orchestrator.service.ccdcallbackhandler.CcdCallbackDtoCreator;
+
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatObject;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.BDDMockito.given;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+
+class CCDUpdateServiceTest {
```

Comment:
Hi @mdayican ,
As discussed on call, could you do the following (follow the convention)?

1. Make/Mark this Test class as public.
2. Use @RunWith(MockitoJUnitRunner.class) and Mock the objects via @Mock.

Reply:
Mock annotation is not used because `CcdCallbackDtoCreator` is not mocked. do you want it to mocked?

Reply:
Yes please.

Reply:
test are updated and done refactoring. Retesting may needed.